apps/passwd.c

```diff
@@ -47,6 +33,14 @@ static unsigned const char cov_2char[64] = {
     0x73, 0x74, 0x75, 0x76, 0x77, 0x78, 0x79, 0x7A
 };
 
+static const char ascii_dollar[] = { 0x24, 0x00 };
+#if defined(CHARSET_EBCDIC) && !defined(CHARSET_EBCDIC_TEST)
+static const char ebcdic_dollar[] = { 0x5B, 0x00 };
+# define dollar ebcdic_dollar
+#else
+# define dollar ascii_dollar
+#endif
```

Comment:
However! While this is an admirable effort, one probably should keep in mind that EBCDIC support is effectively defunct. And suggestion still is that if we don't get any kind of access(\*), we would be probably better off scarping it. Because it's more appropriate to *not* provide an illusion of something that isn't there.

(\*) Question effectively is about z/OS, as the only[?] one alive EBCDIC-based system. But note that we don't have any z/OS configs. And we can be pretty confident that those that are there were never tested, at least not with 1.1. So that at least those configs should be scraped, in which case we won't have any EBCDIC configs...